src/3d/shaders/pointcloud.frag

```diff
@@ -81,33 +81,31 @@ vec3 exactColorRamp()
     vec3 color=colorRampLine.yzw;
     float value=colorRampLine.x;
     if ( abs( parameter - value ) < 0.01 )
-      return color;
+      return vec4( color, 1.0f );
   }
-
-  return vec3(0.5, 0.5, 0.5);
+  return vec4(0.0, 0.0, 0.0, 0.0f);
```

Comment:
This is a bit confusing to return a color with zero alpha if we're not having alpha blending enabled... Maybe let's keep alpha=1 to avoid us getting confused why it's like that at some point later :-)